docs/release-notes/README.md

```diff
@@ -87,6 +87,33 @@ classifying the release note according to one of the following categories:
        * internal project status or plans
        * Jira issue or PR identifiers
 
+
+   * Use double backticks not single backticks for inline code samples.
+
+     * Incorrect
+
+     ```markdown
+     The `foo(bar)` endpoint no longer accepts `waldo` and now accepts a `garply`.
+     ```
+
+     * Correct
+
+     ```markdown
+     The ``foo(bar)`` endpoint no longer accepts ``waldo`` and now accepts ``garply``.
+     ```
```

Comment:
```suggestion
       ```markdown
       The ``foo(bar)`` endpoint no longer accepts ``waldo`` and now accepts ``garply``.
       ```
```